lpvartypes.pas

```diff
@@ -1827,7 +1843,18 @@ function TLapeType.Eval(Op: EOperator; var Dest: TResVar; Left, Right: TResVar;
         LapeExceptionFmt(lpeIncompatibleOperator1, [LapeOperatorToString(op), AsString])
       else
         LapeExceptionFmt(lpeIncompatibleOperator, [LapeOperatorToString(op)]);
-
+    except
+      on e: Exception do
+      begin
+        Dest := NullResVar;
+        Result := TryOperatorOverload();
```

Comment:
That would indeed be more clear. The TLapeTree_Operator is a good fit for EvalRes, so i'd wanna keep it in there. Which means moving the Eval to TLapeTree_Operator.